### PR TITLE
Imports

### LINKED ISSUE #24: imports (`use`)

### PR DESCRIPTION
**Pull request by nrc**
Closes #24

### CODE REVIEW

File: guide/items.md [hunk 1]
```diff
@@ -433,4 +433,130 @@ before:
...
+```rust
+use a::b::c;
+use a::b::d::*;
+use a::b::{foo, bar, baz};
+```
+
+
+#### Large list imports
+
+Prefer to use multiple imports rather than a multi-line import. However, tools
```
Comment (cramertj) [inline guide/items.md]: What does `prefer` mean here, if tools aren't supposed to enforce it? I thought the focus of these RFCs was to determine how Rust formatting tools behaved.

File: guide/items.md [hunk 2]
```diff
@@ -433,4 +433,130 @@ before:
...
+should not split imports by default (they may offer this as an option).
+
+If an import does require multiple lines, then break after the opening brace
```
[the remark this answers is not in the document]
Comment (joshtriplett) [reply guide/items.md]: Reasonable point, and I agree.

File: guide/items.md (hunk 1 above)
Comment (nrc) [reply guide/items.md]: That is the focus, but we have also given advice in some cases for humans to follow where tools can't reformat

File: guide/items.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nrc) [reply guide/items.md]: Yes, this was my intention; I'll clarify.

File: guide/items.md (hunk 1 above)
Comment (cramertj) [reply guide/items.md]: Sounds good! Thanks for the explanation.

**Pull request merged**